src/components/shared/UserMention/UserMention.tsx

```diff
@@ -37,21 +39,18 @@ const UserMention = ({
   ...props
 }: Props) => {
   const fallbackTo = to || `/user/${username}`;
-  const popoverProps: Partial<InfoPopoverProps> = {
-    popperOptions,
-    trigger: showInfo ? 'click' : 'disabled',
-    showArrow: popperOptions && popperOptions.showArrow,
-  };
-
-  const { data: userAddressData } = useUserAddressQuery({
-    variables: {
-      name: username || '',
-    },
-  });
-
-  const { data } = useUserQuery({
-    variables: { address: userAddressData?.userAddress || '' },
-  });
+  const trigger = showInfo ? 'click' : 'disabled';
+  const showArrow = popperOptions && popperOptions.showArrow;
+
+  // const { data: userAddressData } = useUserAddressQuery({
+  //   variables: {
```

Comment:
Why do we have the commented out code?

Reply:
The plan was to leave this in place for to make it easier for a separate task that would update UserMention functionality